install_config/master_node_configuration.adoc

```diff
@@ -1377,6 +1377,10 @@ with the 'gquota' option, and the matching security context contraint's fsGroup
 type set to 'MustRunAs'.
 ====
 
+The node configuration file determines the resources of a node. See the
```

Comment:
maybe s/See..../You can xref[allocate the node's resources] to specific components.